contracts/jobs/JobsManager.sol

```diff
@@ -263,8 +263,7 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         uint256 _claimId,
         uint256 _segmentNumber,
         string _dataStorageHash,
-        bytes32 _dataHash,
-        bytes32 _transcodedDataHash,
+        bytes32[2] _dataHashes,
```

Comment:
Grouped `dataHash` and `transcodeDataHash` into a single fixed array parameter to get around stack depth too deep issues...